corehq/apps/fixtures/utils.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+
+def clean_fixture_field_name(field_name):
+    """Effectively slugifies a fixture's field name so that we don't send
+    bad XML back from the phone. Ideally, the fixture name should be
+    verified as a good slug before using it.
+    """
+    pattern = '([/\\<> ])'
+    return re.sub(pattern, '_', field_name)
```

Comment:
I think these field names need to also be prevented from starting with `"xml"`.  That's against the xml spec, although I don't know whether or not that'll actually fail hard in CommCare.